### ISSUE TITLE
Disable cc_uploader consul support

### ISSUE THREAD

**Issue by wildloop** [cloudfoundry/capi-release#138]
## Issue

cf-deployment use bosh-dns instead of consul, but there is no way to disable cc_uploader consul support and it causes errors:
```
cc_uploader rs2 - [instance@47450 director="" deployment="cf" group="api" az="z1" id=""] {"level":"error","source":"cc-uploader","message":"cc-uploader.registration-runner.failed-registering-service","data":{"error":"Put http://127.0.0.1:8500/v1/agent/service/register: dial tcp 127.0.0.1:8500: connect: connection refused","service":"cc-uploader","session":"5"}}
```

## Context

https://github.com/cloudfoundry/capi-release/blob/develop/jobs/cc_uploader/templates/cc_uploader_config.json.erb#L17: `config[:consul_cluster] = "http://127.0.0.1:#{p("capi.cc_uploader.consul_agent_port")}"`

8500 it's default value: https://github.com/cloudfoundry/capi-release/blob/develop/jobs/cc_uploader/spec#L68 

## Steps to Reproduce

Default cf-deployment

## Expected result

Flag to disable cc_uploader consul support

**Comment by sethboyles** [comment]
Thanks for submitting this issue!  We have a story to take care of this in our backlog: https://www.pivotaltracker.com/story/show/161135879.

@ssisil , maybe we should prioritize this soon?  It creates a lot of noise in the `cc-uploader` logs.

@sunjayBhatia @heyjcollins Do you know when consul support will be removed from Diego entirely?

**Comment by sunjayBhatia** [comment]
By default Diego components have stopped using consul altogether and all the codepaths/options that use/enable consul will be removed in diego-release v3.0.0 (date TBD)

**Comment by mxplusb** [comment]
@sunjayBhatia do you know when v3 is going to be released? I have a ton of these log messages and I've trying to cipher through them vs real Consul errors.

**Comment by sunjayBhatia** [comment]
Cc @cloudfoundry/cf-diego ^^

**Comment by tcdowney** [comment]
We ended up removing Consul support from cc-uploader in June (see https://github.com/cloudfoundry/cc-uploader/commit/3f212b95bccf379ec00f50fdf31a8d6d190e2119). This probably shipped in capi-release 1.84.0 or 1.85.0. Someone from @cloudfoundry/cf-capi could confirm the exact version.

You shouldn't see these log lines anymore in a version newer than those.
